issue_36.md

```diff
@@ -0,0 +1,32 @@
+# Segment recordings by runs
+
+#36
+Ideas for name : split4phys, split4runs, obj2split,  split4bids, rec2run, phys2runs, ses2run, 4run2split
+
+# The option we settled for :
+Integrate the splitting function with phys2bids.
+
+# What we need to do
+## 1.Adapt phys2bids parser arguments, in `run.py`
+- [ ] ntp type, from int to list
+- [ ] tr type, from float to list
+- [ ] (uncertain) either add argument to specify the file is multi-run, or detect if list contains multiple item.
```

Comment:
I think that a great way to go would be detect if len(list) > 1 - in which case we know we need to call for the split function!